site/docs/numbers/lookupNumbers.mdx

```diff
@@ -154,15 +224,6 @@ Content-Type: application/json
 </TabItem>
 <TabItem value="one_tn_complete">
```

Comment:
labels should be "COMPLETE - One TN", "COMPLETE - Multiple TNs", "COMPLETE - No Information", "PARTIAL_COMPLETE - Multiple TNs" 

In other words, please add the status before the additional info :)